docs/gcloud-config.rst

```diff
@@ -0,0 +1,55 @@
+Configuration
+*************
+
+Overview
+========
+
+- Use service :class:`Client <gcloud.client.Client>` objects to configure
+  your applications.
+
+  For example:
+
+  .. code-block:: python
+
+     >>> from gcloud import bigquery
+     >>> client = bigquery.Client()
+
+- :class:`Client <gcloud.client.Client>` objects hold both a ``project``
+  and an authenticated connection to a service.
+
+- The authentication credentials can be implicitly determined from the
+  environment or directly via
+  :meth:`from_service_account_json <gcloud.client.Client.from_service_account_json>`
+  and
+  :meth:`from_service_account_p12 <gcloud.client.Client.from_service_account_p12>`.
+
+- Logging in with the `Google Cloud SDK`_ will automatically configure a JSON
```

Comment:
Is there a link to use here for "logging in". The [CLI docs][1] aren't that user friendly (it's just a web version of `--help`), but maybe there is a better one that shows what "logging in" is?

[1]: https://cloud.google.com/sdk/gcloud/reference/auth/login